source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,203 @@
+      the destination cluster. When you add a set of databases to the
+      filter, you also exclude any other databases in the cluster.
+      
+      If specify a collection in your filter, you also exclude any other
+      collections that are in the same database.
+
+   .. step:: Create a Filter.
+
+      The ``includeNamespaces`` option specifies a filter that you can
+      pass to the :ref:`/start <c2c-api-start>` API. 
+
+      The filter syntax is: 
+
+      .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+
+      Create an entry in the ``includeNamespaces`` array for each
+      database that you identified in step 1. Use the ``"database"``
+      field to specify the database name.
+      
+      If you want to filter on collections within a database, add those
+      collections to the ``"collections"`` field for that database
+      entry.
+
+   .. step:: Use the Filter.
+
+      To use the filter, attach the filter document when you make the
+      :ref:`/start <c2c-api-start>` API call to begin syncing.
+
+      .. literalinclude:: /includes/api/requests/start-filtered.sh
+         :language: shell
+
+For an example configuration, see: :ref:`c2c-example-start-with-filter`.
+
+.. _c2c-change-filter:
+
+Change the Filter
+-----------------
+
+You cannot update an existing filter. You must remove the old filter and
+restart ``mongosync`` with a new one. 
+
+When ``mongosync`` ran your original filter, it created user databases
+and the ``mongosync_reserved_for_internal_use`` system database on the
+destination cluster. You must remove those databases before restarting
+``mongosync`` with your new filter.
+
+Follow these steps to prepare the destination cluster for a new filter.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Remove ``mongosync_reserved_for_internal_use``.
+
+      a. Stop the ``mongosync`` process. 
+      
+      #. Connect to the :binary:`mongos` instance in the destination
+         cluster with :binary:`mongosh`.
+
+      #. Drop the ``mongosync_reserved_for_internal_use`` system
+         database.
+
+         .. code-block:: javascript
+          
+            use mongosync_reserved_for_internal_use
+            db.dropDatabase()
+
+   .. step:: Remove user databases.
+
+      a. List the databases in the cluster
+
+         .. code-block:: javascript
+          
+            show databases
+
+      #. Remove user databases. The ``admin`` and ``config`` databases
+         are system databases. If the ``show databases`` command lists
+         any other databases on the destination cluster, you must remove
+         them.
+
+         Repeat this step for each user database list:
+
+         .. code-block:: javascript
+          
+            use <user database name>
+            db.dropDatabase()
+            db.dropDatabase()
+
+         Note: After the first :method:`db.dropDatabase()` operation
+         completes, you need to run it a second time to remove the
```

Comment:
I'm not familiar with this. I typically only drop the database once, and it works.